src/dev-app/example/example-module.ts

```diff
@@ -9,14 +9,14 @@
 import {CommonModule} from '@angular/common';
 import {NgModule} from '@angular/core';
 import {MatExpansionModule} from '@angular/material/expansion';
-import {ExampleModule as ExampleDataModule} from '@angular/material-examples';
+import {ExampleModule as DocsExampleModule} from '@angular/examples';
```

Comment:
While being at it, I renamed `material-examples` to `examples`. I think there is nothing wrong with doing that in the context of our repository. We could also use `@angular/components-examples`, but that would be inconsistent with the folder name then. I'd like the folder name to be just called `src/examples`.. thoughts?

Reply:
I actually want to split this up so that we have `material-examples` and `cdk-examples` since we're going to move the cdk docs to angular.io (obviously not strictkly related to this PR). I also don't like having imports like `@angular/examples` since it's not as specific to components.

Reply:
Yeah, I figured that we want to split this into separate packages at some point, but yeah this is probably something for a different PR. 

Yeah, I'm not too happy about `@angular/examples` either, but in the context of this repository, I could imagine it being okay? Otherwise I'm okay renaming to `components-examples` for now (as well as the folder). What do you think?

Reply:
`components-examples` is fine as a short-term thing

Reply:
Updated to `components-examples` for now.